Gruntfile.js

```diff
@@ -1406,6 +1407,19 @@ module.exports = function(grunt) {
 		}
 	} );
 
+	grunt.registerTask( 'blockJson2PHP', 'Copies block.json file contents to block-json.php.', function() {
+		var blocks = {};
+		grunt.file.recurse( SOURCE_DIR + 'wp-includes/blocks', function( abspath, rootdir, subdir, filename ) {
+			if ( /^block\.json$/.test( filename ) ) {
+				blocks[ subdir ] = grunt.file.readJSON( abspath );
+			}
+		} );
+		grunt.file.write(
+			SOURCE_DIR + 'wp-includes/blocks/blocks-json.php',
+			'<?php return ' + json2php( blocks ) + ';'
```

Comment:
Should end the PHP tag.
```suggestion
			'<?php return ' + json2php( blocks ) + '; ?>'
```

Reply:
The tag was left open on purpose 😄 
PHP doesn't have an issue with open PHP tags at the end of a file - in fact it's usually recommended and that's what all PHP files in WP core do.
Tags close at the end of a file only when that file is a template file. In all other cases, we run the risk of a `Headers already sent` error in case a whitespace character is added at the end of the file after the closing `?>` tag.

Reply:
Thanks for the clarification Ari.